### ISSUE TITLE
[bug] mldocs.py crashes due to lack of python dependencies

### ISSUE THREAD

**Issue by atttx123** [lsgrep/mldocs#14]
### behavior: 
![image](https://user-images.githubusercontent.com/1284339/105669837-fa6a7000-5f1a-11eb-9bc0-db9528b63dff.png)
![image](https://user-images.githubusercontent.com/1284339/105669854-022a1480-5f1b-11eb-99fd-28c3e47bbe87.png)

### error log:
```
Traceback (most recent call last):
  File "mldocs.py", line 9, in <module>
    import requests
ImportError: No module named 'requests'
```

### How to fix
1. download [requirements.txt](https://github.com/lsgrep/mldocs/blob/master/requirements.txt)
2. run `python2 -m pip install -r requeirements.txt`

**Comment by lsgrep** [comment]
Hi, Thanks for reporting this issue.  I've created a pull request https://github.com/lsgrep/mldocs/pull/15 to bundle `requests` into the `mldocs` package so that this workflow won't rely on external `requests` package from the host system.

However, could you share your Python2 version? `/usr/bin/python --version`

**Comment by atttx123** [comment]
* OS X version:  10.15.7
* python version: 2.7.16

OR

- OS X version: 10.14
- python version: 2.7.10

**Comment by lsgrep** [comment]
Thanks, I've released the `0.0.4` (https://github.com/lsgrep/mldocs/releases/tag/0.0.4), in which, `mldocs`  won't rely on external `requests` package. You can update the workflow with command `ml workflow:update` .

**Comment by atttx123** [comment]
Thx, i will close this issue
